runtime/src/bank.rs

```diff
@@ -161,6 +161,11 @@ pub struct TransactionResults {
     pub fee_collection_results: Vec<Result<()>>,
     pub processing_results: Vec<TransactionProcessResult>,
 }
+pub struct TransactionBalanceSet {
+    pub pre_balance: TransactionBalances,
```

Comment:
```suggestion
    pub pre_balances: TransactionBalances,
```
(`post_balances` too)